README.md

```diff
@@ -212,16 +212,20 @@ File fields provide details about each file.
 
 ## <a name="geo"></a> Geo fields
 
-Geo fields can carry data about a specific location related to an event or geo information for an IP field.
+Geo fields can carry data about a specific location related to an event or geo information derived from an IP field.
 
 
+The `geo` fields are expected to be nested at: `destination.geo`, `device.geo`, `host.geo`, `source.geo`.
+
+Note also that the `geo` fields are **not** expected to be used directly at the top level.
```

Comment:
Why do we make this bold? I see `geo` as a reserved prefix in ECS so even if it is used on the top level I would expect everything to work.

Reply:
The bold is to make sure the difference between the two versions of this canned sentence sticks out.

And in the meeting about reuseable objects, we agreed which object was expected where, including whether or not it was expected at the top level. We also agreed on the "expectation" part, and it not being an absolute requirement.

So the phrasing "not expected" allows for someone to do it if needed (as they can already do what they want around ECS anyway).

I can get rid of the bold, though. I did end up writing the reverse sentence in a way that doesn't look too much like this one (see [#user fields](https://github.com/webmat/ecs/tree/reuseable-objects#-user-fields)).